### ISSUE TITLE
Document how to integrate with local app users

### ISSUE THREAD

**Issue by arribatec-cloud-1** [tomasvotava/fastapi-sso#127]
Some apps want to provide the option of using single sign on along side the options of using app-spesific local users.

There is **no documentation** on how to solve such a case. Which parts of the auth process is the same, and which is sso spesific? How could a local auth be introduced?

**Comment by tomasvotava** [comment]
I don't think this should be part of fastapi-sso's docs as it is more of a general thing. There are tons of examples in the official docs of [fastapi](https://fastapi.tiangolo.com/tutorial/security/first-steps/). `fastapi-sso` is not authentication and authorization package, it simply makes the negotiation with Open ID provider easier. I might add some complex examples in the future, but it's probably better to just rely on people using it and trying to find some github repos.

Feel free to try to change my mind, though 😄
